src/services/ubi/community.ts

```diff
@@ -628,7 +628,13 @@ export default class CommunityService {
                 active: true,
             },
         });
-        return result.map((r) => r.toJSON() as ManagerAttributes);
+        return result.map((r) => {
+            const manager = r.toJSON() as ManagerAttributes;
+            return {
```

Comment:
By changing this, we have to keep in mind that the frontend will need to know this information. Up until now, the returned information was a `ManagerAttributes` object (see https://github.com/impactMarket/api/blob/master/src/api/routes/community.ts#L659) for reference. As the return structure is changed, the swagger docs need to be updated, so the frontend knows what to update.

Reply:
sure! I added this new information at the swagger, on the Beneficiary and Manager list.